argo_cd/assets/configuration/spec.yaml

```
@@ -0,0 +1,20 @@
+name: Argo_CD
+files:
+- name: argo_cd.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/openmetrics_legacy
```

```suggestion
      - template: init_config/openmetrics
```

this should be updated to match